homeassistant/components/songpal/media_player.py

```diff
@@ -221,6 +227,17 @@ def unique_id(self):
         """Return a unique ID."""
         return self._sysinfo.macAddr
 
+    @property
+    def device_info(self):
+        """Return the device info."""
+        return {
+            "identifiers": {(DOMAIN, self.unique_id)},
```

Comment:
Should this info define `connections`, too?

Reply:
We only need either connections or identifiers.